### PR TITLE
Adding usable xblocks to the pip dependencies and cms installed apps

### PR DESCRIPTION
**Pull request by jfavellar90**
@felipemontoya @diegomillan yet again the PR about xblocks

### CODE REVIEW

File: requirements/edx/github.txt [hunk 1]
```diff
@@ -83,7 +83,7 @@ git+https://github.com/edx/edx-val.git@0.0.9#egg=edxval==0.0.9
 git+https://github.com/pmitros/RecommenderXBlock.git@v1.1#egg=recommender-xblock==1.1
 git+https://github.com/solashirai/crowdsourcehinter.git@518605f0a95190949fe77bd39158450639e2e1dc#egg=crowdsourcehinter-xblock==0.1
 -e git+https://github.com/pmitros/RateXBlock.git@367e19c0f6eac8a5f002fd0f1559555f8e74bfff#egg=rate-xblock
--e git+https://github.com/pmitros/DoneXBlock.git@857bf365f19c904d7e48364428f6b93ff153fabd#egg=done-xblock
```
Comment (felipemontoya) [inline requirements/edx/github.txt]: Why are you upgrading this now. [Eucalyptus uses te 857...](https://github.com/edx/edx-platform/blob/open-release/eucalyptus.master/requirements/edx/github.txt)

File: requirements/edunext/custom.txt [hunk 2]
```diff
@@ -19,3 +19,14 @@
...
+# Maintained xblocks from openedx xblocks directory (see https://openedx.atlassian.net/wiki/display/COMM/XBlocks+Directory)
+
+# Active table xblock
+-e git+https://github.com/open-craft/xblock-activetable.git@1786a2c8fc06ca3ecfac1972464048982bd486ff#egg=activetable-xblock
+# Image explorer xblock
+-e git+https://github.com/edx-solutions/xblock-image-explorer.git@31e6da363b3c0c61de75efe06f0c8cc502b544e5#egg=xblock-image-explorer
+# School yourself xblock
+-e git+https://github.com/schoolyourself/schoolyourself-xblock.git@5e4d37716e3e72640e832e961f7cc0d38d4ec47b#egg=schoolyourself-xblock
+# Vector drawing xblock
+-e git+https://github.com/open-craft/xblock-vectordraw.git@v0.2.1#egg=vectordraw-xblock==0.2.1
```
Comment (felipemontoya) [inline requirements/edunext/custom.txt]: Eucalyptus uses v0.2m but 0.2.1 is such a minor change that we can use it like this

**Comment by jfavellar90** [comment]
I've changed the versions of the xblocks to be eucalyptus compatible

**Comment by felipemontoya** [comment]
I think the new changes are fine. :+1:

**Comment by jfavellar90** [comment]
:+1:

**Pull request merged**